src/pages/StudentRegister/components/PersonalDetails.tsx

```diff
@@ -44,8 +47,26 @@ const PersonalDetailsSchema = z.object({
   gender: z.string(),
   address: z.string().min(5).max(150),
   parentName: z.string(),
-  parentContact: z.string().length(10),
-  contact: z.string().length(10),
+  parentContact: z
+    .number({
+      invalid_type_error: "Please enter a valid contact number",
+    })
+    .min(1000000000, {
```

Comment:
Is it number in backend?

Reply:
Yes sir it is defined as number in Student-User-Model. This was also a bug while submitting the form, as earlier it was send with zod a string but in backend we have defined contactand parent contact as Number.

Reply:
We need to change at both places, this number comparison is wrong way of doing it
Make it string everywhere, then just compare the length of the string